stream-chat-android-client/src/main/java/io/getstream/chat/android/client/notifications/handler/MessagingStyleNotificationHandler.kt

```diff
@@ -69,16 +77,18 @@ internal class MessagingStyleNotificationHandler(
             newMessageIntent(message.id, channel.type, channel.id),
             PendingIntent.FLAG_IMMUTABLE or PendingIntent.FLAG_UPDATE_CURRENT
         )
-        val initialMessagingStyle = restoreMessagingStyle(channel) ?: createMessagingStyle(currentUser, channel)
-        val notification = NotificationCompat.Builder(context, getNotificationChannelId())
-            .setSmallIcon(R.drawable.stream_ic_notification)
-            .setStyle(initialMessagingStyle.addMessage(message.toMessagingStyleMessage(context)))
-            .setContentIntent(contentPendingIntent)
-            .addAction(NotificationMessageReceiver.createReadAction(context, notificationId, channel, message))
-            .addAction(NotificationMessageReceiver.createReplyAction(context, notificationId, channel))
-            .build()
-        addNotificationId(notificationId)
-        notificationManager.notify(notificationId, notification)
+        ChatClient.instance().scope.launch {
```

Comment:
Can we pass `scope` via constructor?

Reply:
We can, but it would be needed it to be passed by our customers and won't be the same that our `ChatClient` uses.
The `MessagingStyleNotificationsHandler` is created inside of our `NotificationHandlerFactory` that was created to be used by our customers before the `ChatClient` is created.